rpcs3/Emu/Cell/Modules/cellPngEnc.h

```diff
@@ -166,4 +170,5 @@ struct CellPngEncStreamInfo
 	be_t<u32> streamSize; // usz
 	be_t<u32> processedLine;
 	be_t<u64> userData;
+	// TODO: where are the missing 4 bytes?
```

Comment:
It's an alignment thing, it aligns the size to alignof(T) at the end of the structure to support aligned arrays of the type.

Reply:
I know. I just don't want to bother right now